CefSharp/DevTools/IDevToolsClient.cs

```diff
@@ -20,15 +20,18 @@ public interface IDevToolsClient
         /// </summary>
         event EventHandler<DevToolsEventArgs> DevToolsEvent;
 
+        IDisposable RegisterEventHandler<T>(string eventName, EventHandler<T> eventHandler) where T : DevToolsDomainEventArgsBase;
+
```

Comment:
Please add xml doc. You can add the doc here and use inheritdoc for the implementation.